saleor/core/emails.py

```diff
@@ -4,7 +4,13 @@
 from ..core.utils import build_absolute_uri
 
 
-def get_email_base_context():
-    site = Site.objects.get_current()
+def get_email_bases():
```

Comment:
Let's add a docstring here.

Reply:
I think `get_email_context` would be a quite good name for this function, although it returns a tuple. But maybe with a good docstring, it would be good enough.